concepts.wip/booleans/introduction.md

```diff
@@ -2,3 +2,37 @@
 
 True or false values are represented by the `Bool` type.
 It contains only two values: `true` and `false`.
+
+In Julia, it is a built-in _data_ _type_.
+
+> In programming languages with a built-in Boolean data type, such as Pascal and Java[^1], the comparison operators such as `>` and `!=` are usually defined to return a Boolean value[^2].
+
+[^1]: This also includes Julia, among other programming languages.
+[^2]: [Programming generalities](https://en.wikipedia.org/wiki/Boolean_data_type#Generalities).
```

Comment:
Imho it's not relevant if it's built-in or not in Julia, and I haven't seen the notion of "data types" being used that often. It's just a type.

Could you explain the reasoning behind adding the comparison to Pascal or Java? From Julia user surveys we know many people come from Python, Matlab, C/C++, so a comparison to those seems more fitting. I'd suggest removing this paragraph entirely otherwise.

Reply:
I see your point. I'll have it removed. Thanks.